vdev/src/commands/cli.rs

```diff
@@ -0,0 +1,56 @@
+use clap::{Parser, Subcommand};
+use clap_verbosity_flag::{InfoLevel, Verbosity};
+
+use crate::app::Application;
+use crate::commands;
+
+/// Vector's unified dev tool
+#[derive(Parser, Debug)]
+#[
+    command(
+        bin_name = "vdev",
+        author,
+        version,
+        about,
+        disable_help_subcommand = true,
+        long_about = None,
+    )
+]
+pub struct Cli {
+    #[clap(flatten)]
+    pub(crate) verbose: Verbosity<InfoLevel>,
+
+    #[command(subcommand)]
+    command: Commands,
+}
+
+#[derive(Subcommand, Debug)]
+enum Commands {
+    /// Build Vector
+    Build(commands::build::Cli),
+    /// Manage the config file
+    Config(commands::config::cli::Cli),
+    /// Collection of useful utilities
+    Meta(commands::meta::cli::Cli),
+    /// Run a command within the repository
+    Run(commands::run::Cli),
+}
+
+impl Cli {
+    pub fn exec(&self, app: &Application) {
+        match &self.command {
+            Commands::Build(cli) => {
+                cli.exec(&app);
+            }
```

Comment:
What does `clippy` say if you drop the semicolon? It would make these stanzas much more compact:
```suggestion
            Commands::Build(cli) => cli.exec(&app),
```

Reply:
Fixed!